### PR TITLE
fix image repo in readme

### PR DESCRIPTION
**Pull request by msau42**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ for more details.
 ## Project Status
 Status: Alpha
 
-Latest image: `gcr.io/google-containers/volume-csi/gcp-filestore-csi-driver:v0.2.0`
+Latest image: `gcr.io/gke-release/gcp-filestore-csi-driver:v0.2.0`
```
Comment (davidz627) [inline README.md]: can use: `gke.gcr.io`

"The vanity domain 'gke.gcr.io' provides the closest mirror available for the request. If the request is from EU it should serve an image from 'eu.gcr.io/gke-release/'."

**Comment by davidz627** [comment]
/lgtm
some weird verify failures

**Comment by msau42** [comment]
Pushed a commit to fix govet errors. Unclear why it's only starting to fail now

**Pull request merged**